src/dynarec/arm64/dynarec_arm64_00.c

```diff
@@ -3082,6 +3083,7 @@ uintptr_t dynarec64_00(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int nin
         case 0xE9:
         case 0xEB:
             BARRIER(BARRIER_MAYBE);
+            if (box64_dynarec_test == 2) { NOTEST(x1); }
```

Comment:
mmm, NOTEST on JMP defeat the purpose of thest. You should NOTEST only if the jump is outside the block.

Reply:
the issue is I can't know if the jump is outside or not from the interpreter..

Reply:
All this seems very complex, and with a lot of function signature changes. if NOTEST(x1) only does runtime changes, that's fine.

Reply:
Yeah, that’s what it takes :(